tests/ssl-opt.sh

```diff
@@ -6987,6 +6987,30 @@ run_test    "Force an ECC ciphersuite in the server side" \
             -c "found supported_point_formats extension" \
             -s "server hello, supported_point_formats extension"
 
+requires_config_enabled MBEDTLS_AES_C
+requires_config_enabled MBEDTLS_CCM_C
+requires_config_enabled MBEDTLS_SHA256_C
+requires_config_enabled MBEDTLS_KEY_EXCHANGE_ECDHE_ECDSA_ENABLED
```

Comment:
Suggestion for improvement: I think the list of requirements could be shortened by using `requires_ciphersuite_enabled`. But the list is functionally fine as it is.

Reply:
Done